script.js

```diff
@@ -65,3 +93,6 @@
 //try to solve this recursively
 
 /*code here*/
+function nFactorial(num){
```

Comment:
Try working through our JS mini bootcamp.  We cover recursion and some simple recursive problems.  `nFactorial` is one of the "classic" first recursive problems.  This is an important concept and we go over it more the first week of class.